server/lib/nl_data_spec.py

```diff
@@ -88,7 +88,9 @@ def _highlight_svs(sv_df):
 
 
 def _sample_child_place(main_place_dcid, contained_place_type):
-  # Find a sampled child place
+  """Find a sampled child place"""
+  if not contained_place_type:
+    return
```

Comment:
return None here and also in the end of the function?

Reply:
done!